skrf/circuit.py

```diff
@@ -1656,10 +1773,19 @@ def calculate_ports(cnx: list[tuple[Network, int]]) -> int:
 
     # Generate the connected network and the original port index
     ntwk_cnt = Network()
+
+    # Get the Networks' names
+    name_cnt, name_a, name_b = "", str(ntwkA.name), str(ntwkB.name)
```

Comment:
assignment to `name_cnt` is not necessary I assume?

Reply:
I misunderstood variable scope in Python if statements. No need to assign `ntwk_cnt`, as you mentioned.